### PR TITLE
Quick device menu

### PR DESCRIPTION
**Pull request by mail2bishnoi**
Hi @tananaev 

I have started working on the device quick menu PR. Please review the initial draft.

Can you please tell me how to pass the event(which we pass to onMarkerClick function), on clicking the list item to show the quick menu?

### CODE REVIEW

File: modern/public/styles.css [hunk 1]
```diff
@@ -10,3 +10,7 @@ canvas {
 .mapboxgl-ctrl {
   margin: 10px;
 }
+
+.maplibregl-popup {
```
Comment (tananaev) [inline modern/public/styles.css:14]: Does it not adjust automatically based on the content?

File: modern/src/map/StatusView.js [hunk 2]
```diff
@@ -1,69 +1,141 @@
...
+  },
+  green: {
+    color: theme.palette.common.green,
+  },
+  red: {
+    color: theme.palette.common.red,
+  },
+  gray: {
+    color: theme.palette.common.gray,
+  },
```
Comment (tananaev) [inline modern/src/map/StatusView.js:35]: Should we extract this common part somewhere?

**Review by tananaev — COMMENTED** [review]
Overall looks good, I think. Can you please also attach a screenshot.

> Can you please tell me how to pass the event(which we pass to onMarkerClick function), on clicking the list item to show the quick menu?

Maybe through props? Why do you need to pass the event?

**Comment by tananaev** [comment]
Also don't forget to rebase. You have a merge conflict.

File: modern/public/styles.css (hunk 1 above)
Comment (mail2bishnoi) [reply modern/public/styles.css:14]: No. There is an inline style with a max-width of 240px.

File: modern/src/map/StatusView.js (hunk 2 above)
Comment (mail2bishnoi) [reply modern/src/map/StatusView.js:36]: Should I move these classes to global css file?

**Comment by mail2bishnoi** [comment]
> 
> 
> Overall looks good, I think. Can you please also attach a screenshot.
> 
> > Can you please tell me how to pass the event(which we pass to onMarkerClick function), on clicking the list item to show the quick menu?
> 
> Maybe through props? Why do you need to pass the event?

To show the StatusView on clicking device in the list.

**Comment by tananaev** [comment]
> To show the StatusView on clicking device in the list.

I see what you mean. I think the best option is to tell map that a device needs to selected. Then map can get coordinates to display popup.

File: modern/src/map/StatusView.js (hunk 2 above)
Comment (tananaev) [reply modern/src/map/StatusView.js:36]: I think just a separate file. You can the include it in the object using deconstruction like `...colors`.

File: modern/public/styles.css [hunk 3]
```diff
@@ -10,3 +10,7 @@ canvas {
...
+  max-width: 330px!important;
```
Comment (tananaev) [inline modern/public/styles.css:15]:
```suggestion
  max-width: 330px !important;
```

**Comment by mail2bishnoi** [comment]
Hi @tananaev 

Please review now.

**Comment by tananaev** [comment]
Merged, thank you.

**Pull request merged**